src/components/Amp.js

```diff
@@ -0,0 +1,39 @@
+import "./css/Amp.css";
+
+export default function Amp({ attack, decay, onChangeAttack, onChangeDecay }) {
+  function handleAttackChange(event) {
+    onChangeAttack(Number(event.target.value));
+  }
+
+  function handleDecayChange(event) {
+    onChangeDecay(Number(event.target.value));
+  }
+
+  return (
+    <div className="Function-Board">
+      <div className="Amp-bar">
+        <h2>SET AMP ENVELOPE</h2>
+      </div>{" "}
+      <label className="Amp-label"> ATTACK</label>
```

Comment:
In general, I'd advise to use the `for` attribute for labels. Keep in mind it's called `htmlFor` in React though
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label
Not sure if you want to add this in this PR, since you probably have some more labels in your app

Reply:
ok, I'll check